### ISSUE TITLE
DOMPoint arguments length

### ISSUE THREAD

**Issue by yckart** [trusktr/geometry-interfaces#5]
Initializing `DOMPoint` with 0, 2 or 3 arguments throws `TypeError`.

Seems to be ok to have (at least) 2 arguments:

https://www.w3.org/TR/geometry-1/#DOMPoint (*Example 1*)

```js
var point = new DOMPoint(5, 4);
var matrix = new DOMMatrix(2, 0, 0, 2, 10, 10);
var transformedPoint = point.matrixTransform(matrix);
```

https://www.w3.org/TR/geometry-1/#DOMQuad (*Example 2*)

```js
var point = new DOMPoint(2, 0);
var quad1 = new DOMQuad(point, {x: 12, y: 0}, {x: 2, y: 10}, {x: 12, y: 10});
```

**Comment by trusktr** [comment]
Interesting.

https://www.w3.org/TR/geometry-1/#dom-dompointreadonly-dompointreadonly doesn't mention that, currently it describes passing in the four arguments (x,y,z,w) or a DOMPointInit. 

However I do see the examples now, which I overlooked. Good catch Yannick!

The spec needs some improvement to explicitly describe which arguments are optional.

If you make a PR I will merge! 😃  Otherwise I'll get to this soon as I can.

**Comment by trusktr** [comment]
@yckart Thanks for the report! @PaulCapron fixed it in #8.
